pkg/cmd/validate.go

```diff
@@ -159,16 +160,37 @@ func (c *commandFlags) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	files, err := utils.FindFiles(args...)
-	if err != nil {
-		return err
+	var files []*os.File
+	hasVisited := false
+	for _, arg := range args {
+		if arg == "-" {
+			if !hasVisited {
+				files = append(files, cmd.InOrStdin().(*os.File))
+				hasVisited = true
+			}
+		} else {
+			fileNames, err := utils.FindFiles(arg)
+			if err != nil {
+				return err
+			}
+			for _, fileName := range fileNames {
+				f, err := os.Open(fileName)
```

Comment:
Storing open file handles like this may result in having open an excessively large number of them(for users who are validating large numbers of files)